libs/src/sdk/services/DiscoveryNodeSelector/generateDefaultBootstrapServices.ts

```diff
@@ -0,0 +1,54 @@
+import { EthWeb3Manager } from '../../../services/ethWeb3Manager'
+import { EthContracts } from '../../../services/ethContracts'
+import { IdentityService } from '../../../services/identity'
```

Comment:
This file I _think_ is the only place in SDK where we still depend on libs!

I don't consider that a goal but I think it'd be nice to have that separation while we migrate